gammapy/spectrum/models.py

```diff
@@ -2,24 +2,90 @@
 """Spectral models for Gammapy.
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
+from ..extern.bunch import Bunch
+from . import CountsSpectrum
+import numpy as np
+
 
 __all__ = [
     'SpectralModel',
     'PowerLaw',
     'ExponentialCutoffPowerLaw',
 ]
 
+# Note: Consider to move stuff from _models_old.py here
 
 class SpectralModel(object):
     """Spectral model base class.
     """
+    def __call__(self, energy):
+        kwargs = self.parameters
+        kwargs.update(energy=energy)
+        return self.evaluate(**kwargs)
+
+    def predicted_counts(self, livetime, aeff, edisp):
```

Comment:
@joleroi - Yes, sounds good.

Alternatively, the counts `n_on` and `n_off` could be optional (default: None) on SpectrumObservation construction.
And then @jjlk could also just create a `SpectrumObservation` to compute `npred`.

Also .. why does the [SpectrumObservation](http://docs.gammapy.org/en/latest/api/gammapy.spectrum.SpectrumObservation.html) `__init__` not take `livetime`?

I see the attribute is called `exposure` ... I'd prefer `livetime`, `exposure` sometimes means `A * T` and could be confusing.